### PR TITLE
doc/rgw: instructions for changing multisite master zone

### PR DESCRIPTION
**Pull request by cbodley**
preview should be available here: http://docs.ceph.com/docs/wip-rgw-doc-changing-master/radosgw/multisite/

### CODE REVIEW

**Comment by theanalyst** [comment]
is this a change we introduced recently or did we historically support metdata sync between zgs?

**Comment by cbodley** [comment]
> is this a change we introduced recently or did we historically support metdata sync between zgs?

it was part of the multisite v2 design, it just hasn't seen much testing

File: doc/radosgw/multisite.rst [hunk 1]
```diff
@@ -782,5 +790,62 @@ system's init system, for eg::
...
+   promoting it to master is recommended.
+
+   Similarly, if changes to metadata are being processed by the current master
+   zone while another zone is being promoted to master, those changes are
+   likely to be lost. To avoid this, shutting down any ``radosgw`` instances
+   on the previous master zone is recommended. After promoting another zone,
+   its new period can be fetched with ``radosgw-admin period pull`` and the
+   gateway(s) can be restarted.
+
+To promote a zone (for example, zone ``us-2`` in zonegroup ``us``) to master
```
Comment (yehudasa) [inline doc/radosgw/multisite.rst]: to meta master

**Comment by cbodley** [comment]
updated, thanks. it looks like the doc builders aren't working. i ran `ceph/admin/build-doc` and `serve-doc` to preview, and the formatting looks right

**Pull request merged**